src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/inlinestats/columns_after.test.ts

```diff
@@ -6,137 +6,35 @@
  * your election, the "Elastic License 2.0", the "GNU Affero General Public
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
-import { synth } from '../../../..';
-import type { ESQLFieldWithMetadata, ESQLUserDefinedColumn } from '../../types';
+import { Parser } from '../../../..';
+import type { ESQLColumnData } from '../../types';
+import { type ESQLFieldWithMetadata } from '../../types';
 import { columnsAfter } from './columns_after';
 
-// Test data fixtures
-const createPreviousFields = (fields: Array<[string, string]>): ESQLFieldWithMetadata[] =>
-  fields.map(([name, type]) => ({ name, type } as ESQLFieldWithMetadata));
-
 describe('INLINESTATS', () => {
-  const createUserDefinedColumn = (
-    name: string,
-    type: ESQLFieldWithMetadata['type'],
-    location = { min: 0, max: 10 }
-  ): ESQLUserDefinedColumn => ({ name, type, location });
-
-  const createContext = (userDefinedColumns: Array<[string, ESQLUserDefinedColumn[]]> = []) => ({
-    userDefinedColumns: new Map(userDefinedColumns),
-    fields: new Map<string, ESQLFieldWithMetadata>([
-      ['field1', { name: 'field1', type: 'keyword' }],
-      ['count', { name: 'count', type: 'double' }],
-    ]),
-  });
-
-  const expectColumnsAfter = (
-    command: string,
-    previousFields: ESQLFieldWithMetadata[],
-    userColumns: Array<[string, string]>,
-    expectedResult: Array<[string, string]>
-  ) => {
-    const context = createContext(
-      userColumns.map(([name, type]) => [name, [createUserDefinedColumn(name, type as 'keyword')]])
-    );
-    const result = columnsAfter(synth.cmd(command), previousFields, context);
-    const expected = createPreviousFields(expectedResult);
-    expect(result).toEqual(expected);
-  };
-  it('preserves all previous columns and adds the user defined column, when no grouping is given', () => {
-    const previousFields = createPreviousFields([
-      ['field1', 'keyword'],
-      ['field2', 'double'],
+  it('gets the columns after the query', () => {
+    const previousCommandFields: ESQLFieldWithMetadata[] = [
+      { name: 'field1', type: 'double', userDefined: false },
+      { name: 'buckets', type: 'double', userDefined: false }, // should be overwritten
+      { name: '@timestamp', type: 'date', userDefined: false },
+    ];
+
+    const queryString = `FROM a | STATS AVG(field1) BY buckets=BUCKET(@timestamp,50,?_tstart,?_tend)`;
```

Comment:
we are testing stats instead of inlinestats. Why are all the previous tests gone?

Reply:
oh good catch

Reply:
updated in https://github.com/elastic/kibana/pull/233221/commits/74432cbf04ae562f1cc3331714d336732f4cebed

> Why are all the previous tests gone?

I thought this was adequate. Inlinestats now uses stats columns after under the hood which is tested extensively in stats. Not sure though. I could see an argument the other way, too if we were doing black-box testing.